.github/workflows/code-review.yml

```diff
@@ -1,61 +1,51 @@
 ---
-name: Code Review
+name: Code Review - Linting & Link Checks
 
-##########################################
-# Start the job on push for all branches #
-##########################################
 on:
   pull_request:
     branches:
-    - main
+      - main
```

Comment:
Why the extra tab?

Reply:
Linting picked it up in a previous run as a warning. And it's correct this is nicer YAML